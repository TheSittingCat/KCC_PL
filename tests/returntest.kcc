function test() {
    myfun(6);
    return 2
}