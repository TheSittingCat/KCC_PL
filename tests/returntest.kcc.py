def test(): 
 	2.0
main()