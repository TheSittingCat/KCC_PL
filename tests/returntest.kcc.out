def test(): 
 	hello = "hello" 
 	
main()